dask_ml/preprocessing/_encoders.py

```diff
@@ -163,9 +164,13 @@ def _fit(
         X = check_array(
             X, accept_dask_dataframe=True, dtype=None, preserve_pandas_dataframe=True
         )
+        if SK_024:
+            kwargs = dict(force_all_fininte=force_all_finite)
```

Comment:
a type in the key `force_all_fininte`:
`kwargs = dict(force_all_finite=force_all_finite)`